vaadin-date-time-picker-flow-parent/vaadin-date-time-picker-flow/src/main/java/com/vaadin/flow/component/datetimepicker/DateTimePicker.java

```diff
@@ -391,6 +396,123 @@ public String getLabel() {
         return getElement().getProperty("label");
     }
 
+    /**
+     * Set the aria-label of the component to the given text.
+     *
+     * @param ariaLabel
+     *            the aria-label text to set or {@code null} to clear
```

Comment:
It seems that other methods don't explicitly mention `null`. To keep it consistent, I'd suggest not mentioning `null` here as well. 

```suggestion
     *            the value to set as aria-label
```